xla/service/cpu/onednn_matmul_rewriter.cc

```diff
@@ -63,7 +63,7 @@ inline bool CompatibleElementType(const HloInstruction* instr) {
   return element_type == BF16 || element_type == F32 || element_type == F16;
 }
 
-// Type conversion from and to any of F16, BF16 and FP32.
+// Type conversion from and to any of BF16 and FP32.
```

Comment:
I think F16 is deleted by mistake.
```suggestion
// Type conversion from and to any of F16, BF16 and FP32.
```

Reply:
Since we do not need to add converts for F16 support, I removed it from here.